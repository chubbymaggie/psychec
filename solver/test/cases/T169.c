void f()
{
    const struct Type* x = a->b;
    g(a->b);
    g(x);
}
