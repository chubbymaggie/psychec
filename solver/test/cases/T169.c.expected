/* Forward declarations */
typedef  struct TYPE_2__   TYPE_1__ ;

/* Type definitions */
typedef  struct Type const Type ;
struct TYPE_2__ {struct Type const* b; } ;

/* Variables and functions */
 TYPE_1__* a ; 
 int /*<<< orphan*/  g (struct Type const*) ; 

